addons/web/static/src/scss/utilities_custom.scss

```diff
@@ -182,6 +182,13 @@ $utilities: map-merge(
             property: min-width,
             values: 0,
         ),
+        "text": (
+            class: o-text,
```

Comment:
By consistency with the other classes created in this file, could you remove the `o-` prefix ?

Reply:
Removing `o-` prefix gives css error :pensive: may be because `text` is already used by bootstrap any alternative? :pleading_face:

Reply:
If the key `text` already exists (and it does in regular Bootstrap), it should be extended instead of attempting to override it.

Reply:
Done! Thanks :slightly_smiling_face: